src/styles/app.css

```diff
@@ -170,3 +170,7 @@
   margin-bottom: 1.5rem;
   padding-left: 1rem;
 }
+
+.page-content strong {
+  @apply text-dinum-grey-2;
+}
```

Comment:
we should make it a general rule, and get rid of my sh*tty component `Strong`

Reply:
yeah why not